src/app/cart/page.styles.ts

```diff
@@ -0,0 +1,45 @@
+const styles = {
+  main: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    marginTop: "3.5rem",
+    gap: "2rem",
+    padding: "3rem",
+
+    "@media (max-width: 767px)": {
+      padding: "2rem 1rem",
+    },
+
+    "@media (min-width: 768px) and (max-width: 1023px)": {
+      padding: "2rem 1rem",
+    },
+
+    "@media (min-width: 1024px) and (max-width: 1279px)": {
+      padding: "2rem 7vw",
+    },
+
+    "@media (min-width: 1280px) and (max-width: 1535px)": {
+      padding: "2rem 4vw",
+      marginLeft: "20vw",
+      marginRight: "20vw",
+    },
+
```

Comment:
I don't like how u are using hard-coded media queries, look up the docs of `mui` how to work with media-queries in this ui design system